teachers_digital_platform/search_indexes.py

```diff
@@ -107,3 +107,8 @@ def prepare_council_for_economic_education(self, obj):
 
     def get_model(self):
         return ActivityPage
+
+    def index_queryset(self, using=None):
+        """Only index descendants of the Activity Search page"""
+        search_page = ActivityIndexPage.objects.get()
```

Comment:
@nbucknor thanks for making this change. I wonder if you wouldn't mind adding a comment here mentioning that this logic is safe because ActivityIndexPage is supposed to be a singleton.

Reply:
Will do